ucm2: Add support for MT8192 Asurada Spherion Chromebook

nfraprado opened the pull request:
Add support for the Acer Chromebook 514 CP514-2H, powered by MediaTek Kompanio 820 (MT8192). This machine uses a MT6359 PMIC, with RT1015P as speaker codec and RT5682 as headphone codec.

Signed-off-by: Nícolas F. R. A. Prado <nfraprado@collabora.com>

kholk commented:
On CP214-2H:
`Tested-by: AngeloGioacchino Del Regno <angelogioacchino.delregno@collabora.com>`

nfraprado commented on ucm2/conf.d/mt8192_mt6359_r/mt8192_mt6359_rt1015p_rt5682.conf:
```diff
@@ -0,0 +1 @@
+../../MediaTek/mt8192/mt6359-rt1015p-rt5682/mt8192_mt6359_rt1015p_rt5682.conf
```
Fix sent for the driver name: https://lore.kernel.org/all/20220929205453.1144142-1-nfraprado@collabora.com

I'll update the PR once it gets merged.

nfraprado commented:
Force-pushed v2. Changes:
* Renamed conf.d symlink to follow driver name update
* Removed unnecessary controls in init's bootsequence
* Removed HDMI for now since it isn't yet supported by the kernel
